views/components/Sidebar/SidebarLogo.css

```diff
@@ -0,0 +1,12 @@
+.logo_container {
+  height: 98px;
+  width: 240px;
```

Comment:
we shouldn't need to specify the width

Reply:
[465c369](https://github.com/AMP-SCZ/dpdash/pull/458/commits/465c36952e0f8176957c65d89dfea1c6b86ca296)